frontend/src/components/ActionContent.js

```diff
@@ -20,13 +20,13 @@ const LOADING_ACTION = {
 const checkRequests = {
   sample:    api.samples.template.check,
   container: api.containers.template.check,
-  process:   api.processes.template.check,
+  processSample:   api.processesSamples.template.check,
```

Comment:
Double plural?

Reply:
We can have it single plural (processSamples) if you prefer. I wanted it to be coherent with our API (processes-samples) but if needed, I can change that too, its not too big